src/db.cpp

```diff
@@ -5542,22 +5542,180 @@ void price_cyber(struct obj_data *obj)
       GET_OBJ_AVAILDAY(obj) = 2;
       break;
     case CYB_SKULL:
-      GET_OBJ_VAL(obj, 1) = 0;
-      GET_CYBERWARE_ESSENCE_COST(obj) = 75;
-      if (GET_OBJ_VAL(obj, 3))
+      GET_OBJ_AVAILDAY(obj) = GET_OBJ_AVAILTN(obj) = GET_CYBERWARE_ESSENCE_COST(obj) = GET_OBJ_COST(obj) = 0;
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_OBVIOUS)) {
+        GET_OBJ_COST(obj) = 35000;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 75;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_SYNTHETIC)) {
         GET_OBJ_COST(obj) = 55000;
-      else GET_OBJ_COST(obj) = 35000;
-      GET_OBJ_AVAILTN(obj) = 6;
-      GET_OBJ_AVAILDAY(obj) = 4;
+        GET_OBJ_AVAILTN(obj) = 6;
+        GET_OBJ_AVAILDAY(obj) = 4;
+        GET_CYBERWARE_ESSENCE_COST(obj) = 75;
+      }
+      if (IS_SET(GET_OBJ_VAL(obj, 3), SKULL_ARMORMOD1)) {
+        GET_OBJ_COST(obj) += 6500;
+        GET_OBJ_AVAILTN(obj) = MAX(GET_OBJ_AVAILTN(obj), 8);
+        GET_OBJ_AVAILDAY(obj) = MAX(GET_OBJ_AVAILDAY(obj), 14);
+        GET_CYBERWARE_ESSENCE_COST(obj) += 0;
       break;
     case CYB_TORSO:
-      GET_OBJ_VAL(obj, 1) = 0;
-      GET_CYBERWARE_ESSENCE_COST(obj) = 150;
-      if (GET_OBJ_VAL(obj, 3))
+      GET_OBJ_AVAILDAY(obj) = GET_OBJ_AVAILTN(obj) = GET_CYBERWARE_ESSENCE_COST(obj) = GET_OBJ_COST(obj) = 0;
+      if (IS_SET(GET_OBJ_VAL(obj, 3), TORSO_OBVIOUS)) {
```

Comment:
If you think you might add more bits later, you can save a bit here by having it be `if (!IS_SET(... TORSO_SYNTHETIC)`. Since the cybermod is always either obvious or synthetic, the negation of one means the other is true.

Reply:
This also works neatly around the edge case of a builder forgetting to set either one of these, making the item extremely cheap and zero essence.

Reply:
Yeh, still trying to figure out how to add the other bits to expand upon it, just wanted to nail down the basic foundation to start working off of. Added if (!IS_SET to all of the obvious/synthetic options